api/nakadi-event-bus-api.yaml

```diff
@@ -1203,10 +1204,10 @@ definitions:
         items:
           type: string
         description: |
-          Indicator of the field used for guaranteeing the ordering of Events of this type (used only by 'hash'
+          Indicator of the fields used for guaranteeing the ordering of Events of this type (used only by 'hash'
           PartitionResolutionStrategy).
           If set MUST be a valid required field as defined in the schema.
-          It MUST be set if 'partition_resolution_strategy' is set to 'hash'.
+          It MUST be set if 'partition_resolution_strategy' is set to 'hash'. In other cases it is ignored.
```

Comment:
Yes, I was thinking about that, didn't wanted to alter the original too much. But since you think the same, I change it to make it consistent.